spring-cloud-kubernetes-client-autoconfig/src/test/java/org/springframework/cloud/kubernetes/client/ActuatorDisabledHealthTest.java

```diff
@@ -47,10 +48,15 @@ class ActuatorDisabledHealthTest {
 	private int port;
 
 	@Test
-	void healthEndpointShouldContainKubernetes() {
-		this.webClient.get().uri("http://localhost:{port}/actuator/health", this.port)
+	void healthEndpointShouldNotContainKubernetes() {
+		String response = this.webClient.get().uri("http://localhost:{port}/actuator/health", this.port)
 				.accept(MediaType.APPLICATION_JSON).exchange().expectStatus().isOk().expectBody(String.class)
-				.value(Matchers.not(Matchers.containsString("kubernetes")));
```

Comment:
this test was failing. and it happened because we used to take the json from `/actuator/health` and test if it has any `kubernetes` String inside it. 

Now the most recent changes (I could not track where exactly to be honest), include `diskSpace` with `path` (`path` was not present). So currently it looks like this for example:

```
{
  "status": "UP",
  "components": {
    "discoveryComposite": {
      "description": "DiscoveryClientnotinitialized",
      "status": "UNKNOWN",
      "components": {
        "discoveryClient": {
          "description": "DiscoveryClientnotinitialized",
          "status": "UNKNOWN"
        }
      }
    },
    "diskSpace": {
      "status": "UP",
      "details": {
        "total": 494384795648,
        "free": 116591652864,
        "threshold": 10485760,
        "path": "/Users/wind57/Documents/personal/spring/spring-cloud-kubernetes/spring-cloud-kubernetes-client-autoconfig/.",
        "exists": true
      }
    },
    "livenessState": {
      "status": "UP"
    },
    "ping": {
      "status": "UP"
    },
    "reactiveDiscoveryClients": {
      "description": "DiscoveryClientnotinitialized",
      "status": "UNKNOWN",
      "components": {
        "SimpleReactiveDiscoveryClient": {
          "description": "DiscoveryClientnotinitialized",
          "status": "UNKNOWN"
        }
      }
    },
    "readinessState": {
      "status": "UP"
    },
    "refreshScope": {
      "status": "UP"
    }
  },
  "groups": [
    "liveness",
    "readiness"
  ]
}
```

It does not have `kubernetes` under `components` (which we should have asserted for), but it does have `kubernetes` in general. Again, because that `path` now exists. So our test was failing.